.dockerfiles/install.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+install_deb() {
+	apt-get update
+	apt-get install -y --no-install-recommends \
+		curl=7.64.0-4 \
+		netcat=1.10-41.1 \
+		jq=1.5+dfsg-2+b1
```

Comment:
We should ideally be installing these after Netdata, to ensure they don't interfere with it pulling in the correct dependencies.

Additionally, pinning specific versions is likely to break the install process. Additionally, everything we appear to be doing in the test scripts with these has been stable behavior for many years, so it's not like we need to worry about it changing (I'd actually think we're more likely to hit issues because of pinned versions not being available than due to the interfaces we're using here changing).

Reply:
> We should ideally be installing these after Netdata, to ensure they don't interfere with it pulling in the correct dependencies.

Agreed! I'll change this.

Reply:
> Additionally, pinning specific versions is likely to break the install process. Additionally, everything we appear to be doing in the test scripts with these has been stable behavior for many years, so it's not like we need to worry about it changing (I'd actually think we're more likely to hit issues because of pinned versions not being available than due to the interfaces we're using here changing).

Okay I buy it :) But how do I make Codeacy shut up about this ? :)

Reply:
> Okay I buy it :) But how do I make Codeacy shut up about this ? :)

No idea, I generally just ignore it when it's making recommendations that are situationally stupid like this.

Reply:
Okay 🤣